bots/image-refresh

```diff
@@ -86,15 +86,20 @@ TRIGGERS = {
         "fedora-30/selenium-chrome",
         "fedora-30/selenium-firefox",
     ],
+    "rhel-7-6": [
```

Comment:
Some issues in the commit message:

"3) Of change in bots/ append all external contexts" → "On changes to bots, test all external contexts"
the lone "4.1)" looks weird, just merge it with 4).

"which only checkouts" → "which only checks out". The whole last sentence is hard to understand, how the "... and if" relates to --stable.